sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -183,11 +174,11 @@ def create_queue(self, queue, **kwargs):
         :param queue: The queue name or a `QueueDescription` instance. When it's a str, it will be the name
          of the created queue. Other properties of the created queue will have default values decided by the
          ServiceBus. Use a `QueueDescription` if you want to set queue properties other than the queue name.
-        :type queue: Union[str, QueueDescription]
+        :type queue: Union[str, ~azure.servicebus.management.QueueDescription]
```

Comment:
Is there a reason we put the full path here and not elsewhere?

Reply:
Changed to use full path for all occurrences.